dask/bytes/http.py

```diff
@@ -8,7 +8,7 @@
 DEFAULT_BLOCK_SIZE = 5 * 2 ** 20
 
 
-class HTTPFileSystem(core.FileSystem):
+class HTTPFileSystem(object):
```

Comment:
@martindurant can you verify that this is safe?

Reply:
`core.FileSystem` doesn't define any functionality at all, so yes, safe. We don't check for subclassing of file-systems anywhere.